lib/parliament/utils/config/initializers/timeout.rb

```diff
@@ -1,2 +1,2 @@
 # Temporarily increasing to account for HybridBill document uploads
-Rack::Timeout.timeout = 60
+Rack::Timeout.timeout = 60 if defined?(Rack) && defined?(Rack::Timeout)
```

Comment:
This could be just `if defined?(Rack::Timeout)` - I know I wrote this but still :joy: